docs/community/code_of_conduct.md

```diff
@@ -77,13 +77,13 @@ In cases not involving clear severe and obvious breaches of this code of conduct
 
 3. mediation (if feedback didn’t help, and only if both reporter and reportee agree to this)
 
-4. enforcement via transparent decision (see [Resolutions](https://github.com/napari/napari/blob/main/docs/REPORT_HANDLING_MANUAL.md#resolutions)) by the Code of Conduct Committee
+4. enforcement via transparent decision (see [Resolutions](code_of_conduct_reporting.md#resolutions)) by the Code of Conduct Committee
 
 The committee will respond to any report as soon as possible, and at most within 72 hours.
 
 ## Endnotes
 
 We are thankful to the groups behind the following documents, from which we drew content and inspiration:
 
-* [The SciPy Code of Conduct](https://docs.scipy.org/doc/scipy/reference/dev/conduct/code_of_conduct.html)
+* [The SciPy Code of Conduct](https://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html)
```

Comment:
```suggestion
* [The SciPy Code of Conduct](https://docs.scipy.org/doc/scipy/reference/dev/conduct/code_of_conduct.html)
```

Reply:
@alisterburt your suggested link is broken—for me at least.

Reply:
fixed, thanks! I prefer the public facing link in this case

Reply:
OK. The one I used is the one SciPy uses, but 🤷‍♂️

Reply:
it looks like they move their API docs to github.io so maybe @psobolewskiPhD link is better.

Reply:
Again, it's not **my link**, it's from SciPy Community page:
https://scipy.org/community/